src/ui/hooks/recordings.ts

```diff
@@ -65,6 +65,11 @@ const GET_WORKSPACE_RECORDINGS = gql`
                 name
                 picture
               }
+              comments {
+                user {
+                  id
+                }
+              }
```

Comment:
you could use `comments { id }` here and in `GET_MY_RECORDINGS` instead - that would involve one table less in this query, so it could help with performance (although I didn't see a measurable impact in my tests)

Reply:
I just saw that you need the user's id in #2586